### PR TITLE
Support CUDA 11.6

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

File: .pfnci/config.pbtxt [hunk 1]
```diff
@@ -10,7 +10,7 @@ configs {
     time_limit {
       seconds: 7200
     }
-    command: "sh .pfnci/wheel-linux/main.sh 11.5 3.7"
+    command: "sh .pfnci/wheel-linux/main.sh 11.6 3.7"
```
Comment (kmaehashi) [inline .pfnci/config.pbtxt:13]: Let's keep using 11.5 as CUDA 11.6 driver does not work in CI.

```suggestion
    command: "sh .pfnci/wheel-linux/main.sh 11.5 3.7"
```

**Review by kmaehashi — CHANGES_REQUESTED** [review]

**Comment by kmaehashi** [comment]
Could you add 

```sh
# TODO: revert after confirming CUDA 11.6 driver work on GCP
if [[ "${CUDA}" = "11.6" ]]; then
    export CUPY_RELEASE_SKIP_VERIFY=1
else
    export CUPY_RELEASE_SKIP_VERIFY=0
fi
```

to `.pfnci/wheel-linux/main.sh` so that `build.sh` skips verification steps?

**Comment by kmaehashi** [comment]
Pushed `cupy/cupy-release-tools:cuda-runfile-11.6.0-centos7`.

/test

**Review by kmaehashi — DISMISSED** [review]

File: .pfnci/wheel-linux/main.sh [hunk 2]
```diff
@@ -13,6 +13,13 @@ if [ -z "${BRANCH}" ]; then
     BRANCH="$(cat ./.pfnci/BRANCH)"
 fi
 
+# TODO: revert after confirming CUDA 11.6 driver work on GCP
+if [[ "${CUDA}" = "11.6" ]]; then
+    export CUPY_RELEASE_SKIP_VERIFY=1
+else
```
Comment (kmaehashi) [inline .pfnci/wheel-linux/main.sh:19]: Ah, we should overriding the value set in `config.pbtxt` (for ROCm)

```suggestion
elif [[ "${CUPY_RELEASE_SKIP_VERIFY:-}" != "" ]]; then
```

Comment (takagi) [reply .pfnci/wheel-linux/main.sh]: thoughtful

**Review by kmaehashi — APPROVED** [review]

**Pull request merged**